### PR TITLE
Implements GetStatus

### PR DESCRIPTION
**Pull request by guiMande**
Implementation of GetStatus method, Javadoc and tests. #5

### CODE REVIEW

**Comment by guiMande** [comment]
Do not accept it yet, i forgot something.

**Comment by guiMande** [comment]
Now it's up to date.

File: core-server/src/test/java/org/orbisgis/orbisserver/ITTests/InContainerIT.java [hunk 1]
```diff
@@ -269,7 +269,7 @@ public Result invoke() throws Throwable {
             @Override
             public Result invoke() throws Throwable {
                 return wpsOperationController.displayXML("WPS", "2.0.0", "DescribeProcess",
-                        "file:/C:/Users/mande/AppData/Local/Temp/csvToPointsTable.groovy");
+                        "file:/C:/Users/mande/AppData/Local/Temp/csvToPointsTable.groovy",null);
```
Comment (SPalominos) [inline core-server/src/test/java/org/orbisgis/orbisserver/ITTests/InContainerIT.java]: put the test file in the resources.

**Review by SPalominos — CHANGES_REQUESTED** [review]

File: core-server/src/test/java/org/orbisgis/orbisserver/ITTests/InContainerIT.java (hunk 1 above)
Comment (guiMande) [reply core-server/src/test/java/org/orbisgis/orbisserver/ITTests/InContainerIT.java]: Actually, this is the path which is displayed with GetCapabilities, a better idea will be to remove this test.

Comment (SPalominos) [reply core-server/src/test/java/org/orbisgis/orbisserver/ITTests/InContainerIT.java]: Maybe you could choose a script with a static identifier.

Comment (guiMande) [reply core-server/src/test/java/org/orbisgis/orbisserver/ITTests/InContainerIT.java]: That's what i do with `orbisgis:wps:official:deleteRows`

**Pull request merged**